toolchain/sem_ir/formatter.cpp

```diff
@@ -747,7 +747,18 @@ class Formatter {
       out_ << ' ';
       OpenBrace();
       FormatCodeBlock(impl_info.body_block_id);
-      FormatNameScope(impl_info.scope_id, "!members:\n");
+
+      // Always include the !members label because we always list the witness in
+      // this section.
```

Comment:
```suggestion
      // Print the !members label even if the name scope is empty because we always list the witness in
      // this section.
```

Suggestion to clarify what "Always include" is talking about.